Slim/Schema/Contributor.pm

```diff
@@ -4,20 +4,24 @@ package Slim::Schema::Contributor;
 use strict;
 use base 'Slim::Schema::DBI';
 
-use Scalar::Util qw(blessed);
+use List::Util qw(max);
 
 use Slim::Schema::ResultSet::Contributor;
 
 use Slim::Utils::Log;
 use Slim::Utils::Misc;
 use Slim::Utils::Prefs;
 
+use constant MIN_CUSTOM_ROLE_ID => 21;
```

Comment:
We don't want this value all over the place. Define once and use in helpers below.